### DISCUSSION TITLE
Add jumbotron

### DISCUSSION THREAD

**Discussion by JTSJohnny** [picocss/pico#597] [Ideas]
For my website, i would like a jumbotron like in bootstrreap.

**Comment by ouvreboite** [comment]
Isn't this just a Card ?

From Bootstrap [doc](https://getbootstrap.com/docs/4.0/components/jumbotron/)
![image](https://github.com/user-attachments/assets/04cfee7a-47ea-49da-8073-6f17be44d9d2)

Made with Pico
![image](https://github.com/user-attachments/assets/8ed90877-d8a5-4e68-8dac-ca41b7a2e7ac)

```html
<article>
    <hgroup>
        <h1>Hello, world!</h1>
        <p>This is a simple hero unit, a simple jumbotron-style component for calling extra attention to
            featured content or information.</p>
    </hgroup>
    <hr>
    <p>It uses utility classes for typography and spacing to space content out within the larger container.
    </p>
    <button>Learn more</button>
</article>
```
